src/CycloneDX/CycloneDXSbomCreator.ts

```diff
@@ -302,4 +299,31 @@ export class CycloneDXSbomCreator {
     }
     return undefined;
   }
+
+  private parsePackageJsonName(name: string): Result {
```

Comment:
reply from creator: https://github.com/keithamus/parse-packagejson-name/issues/13#issuecomment-588143151

Reply:
Yeah, super nice of them!